src/modules/jcms_rest/src/Plugin/rest/resource/AnnualReportItemRestResource.php

```diff
@@ -61,8 +63,15 @@ public function get(int $year) : JCMSRestResponse {
         }
       }
 
-      // Image is required.
-      $response['image'] = $this->processFieldImage($node->get('field_image'), FALSE, 'thumbnail', TRUE);
+      // Image is required, for version 1.
+      if ($this->acceptVersion < 2) {
+        if ($image = $this->processFieldImage($node->get('field_image'), FALSE, 'thumbnail', TRUE)) {
+          $response['image'] = $image;
+        }
+        else {
+          throw new JCMSNotAcceptableHttpException('This annual report requires version 2+.');
+        }
```

Comment:
This means no new annual reports should be added until version 1 is not used by Journal anymore, but from the name `annual` I guess there's time for that

Reply:
Since image is still a required field there is no risk of version 1 not being support. I will drop images from `journal-cms` after we have moved `journal` to version 2.